e2e/cypress/integration/search/clear_input_spec.js

```diff
@@ -45,4 +46,39 @@ describe('Search', () => {
         // * The value of the input is empty
         cy.get('#searchBox').should('have.value', '');
     });
+
+    it('MM-T368 - Text in search box should not clear when Pinned posts icon or Saved posts icon is clicked', () => {
```

Comment:
Sorry, I recently [updated the title](https://automation-test-cases.vercel.app/test/MM-T368), should read as
```suggestion
    it('MM-T368 - Text in search box should not clear when Pinned or Saved posts icon is clicked', () => {
```